Api/app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,15 @@
 class SessionsController < ApplicationController
   skip_before_action :require_login, only: :create
 
+  def index
```

Comment:
I think that this functions 'index' and 'show' could be removed, becuase you are using 'Users' controller.